tools/wptrunner/requirements.txt

```diff
@@ -1,7 +1,7 @@
-html5lib == 1.0.1
-mozinfo == 0.10
+html5lib==1.0.1
+mozinfo==0.10
 mozlog==4.0
 mozdebug==0.1.1
-pillow == 5.2.0
+pillow==5.2.0
 urllib3[secure]==1.24.1
-
+requests==2.21.0
```

Comment:
Alternatively, I could change `wptscreenshot.py` or this line:
https://github.com/web-platform-tests/wpt/blob/8a14626934f5748a4ea6210847a02c0d8bbc8560/tools/wptrunner/wptrunner/wptcommandline.py#L11

to do a delayed import (of `requests` or `wptscreenshot`) so that we don't need to install `requests` in `tox`.